pymc3/gp/gp.py

```diff
@@ -136,8 +136,8 @@ def sample_gp(trace, gp, X_values, samples=None, obs_noise=True, model=None, ran
     # Posterior covariance
     S_post = S_zz - tt.dot(tt.dot(S_xz.T, S_inv), S_xz)
 
-    gp_post = MvNormal.dist(m_post, S_post, shape=Z.shape[0])
-    
+    gp_post = MvNormal.dist(m_post, S_post + jitter * tt.eye(S_post.shape[0]), shape=Z.shape[0])
```

Comment:
I think erring on the side of stability as a default is a good idea, so I'm in favor of this. Maybe `chol_const` or something like that?